build.sh

```diff
@@ -1,85 +1,105 @@
-#!/bin/bash
+#!/bin/sh
 
 # Stop on first error
 set -e
```

Comment:
Note that the existing behavior is to stop on error.

Reply:
Yeah, that's why I added a lot more ifs than I would prefer